modules/videojsVideoProvider.js

```diff
@@ -74,13 +74,29 @@ export function VideojsProvider(config, videojs_, adState_, timeState_, callback
       boxingallowed: 1,
       playbackmethod: [ playBackMethod ],
       playbackend: PLAYBACK_END.VIDEO_COMPLETION,
-      skip: 1
+      skip: 1,
+      pos: AD_POSITION.UNKNOWN // default value modified below
     };
 
+    // Placement according to IQG Guidelines 4.2.8
+    // https://cdn2.hubspot.net/hubfs/2848641/TrustworthyAccountabilityGroup_May2017/Docs/TAG-Inventory-Quality-Guidelines-v2_2-10-18-2016.pdf?t=1509469105938
     if (player.isFullscreen()) {
-      // only specify ad position when in Fullscreen since computational cost is low
       video.pos = AD_POSITION.FULL_SCREEN;
     }
+    else if(videojs.dom.findPosition){
+      const {left, top, width, height} = videojs.dom.findPosition(player.el())
```

Comment:
it's best to use vars instead of repeating property chains because property chain don't minify, and vars do. 
i.e. `videojs.dom.findPosition` minifies to `a.dom.findPosition` which is quite lengthy, so repeating that property chain adds a lot of characters.
Instead `const findPosition = videojs.dom.findPosition` minifies to `var b = a.dom.findPosition`, so whenever you use the var, it will be referenced as `b` instead of repeating `a.dom.findPosition`
```
const findPosition = videojs.dom.findPosition;
if (findPosition) {
    const {left, top, width, height} = findPosition(player.el());
```